### PR TITLE
[WBH] Documentation migration of web hosting classic

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+## What happens to my emails?
+
+All your emails and mailboxes will be migrated to cPanel.
+
+</Concept>
+
+<Concept>
+
+## What happens to my email redirections and Transfers?
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:135]:
```suggestion
## What happens to my email redirections and transfers?
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+When will I be billed for the migrated Web Hosting plan?
+
+Your Scalewy Web Hosting billing will start either at the expiration date of Classic Hosting plan or upon the change of the offer on Scalaeway, whichever comes first.
+
+</Concept>
+
+<Concept>
+
+## What about PHP Version Compatibility?
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:168]:
```suggestion
## What about PHP version compatibility?
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+- **Up-to-date system:** cPanel's regular updates contribute to an up-to-date and secure hosting environment. This ensures that your system remains resilient in the face of evolving digital challenges.
+- **A wide range of features:** Explore a range of features offered by cPanel, covering file management, security options, email functionality, and database management. These features collectively form a powerful toolkit to address diverse hosting requirements.
+
+You can initiate the migration of your Web Hosting plan directly from your [Dedibox/Online.net console](https://console.online.net/). For further information, refer to our dedicated documentation [How to migrate an Online.net/Dedibox web hosting plan to Scaleway](/managed-services/webhosting/how-to/migrate-webhosting-plan/).
+
+<Concept opened>
+
+## What will be my new hosting plan?
+
+All migrated hosting will adopt the **migration plan**.
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:29]:
```suggestion
All migrated hosting will transferred to the **migration plan**.
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+Your log files will not be migrated. New log files can be accessed in cPanel at `~/access-logs` or `~/logs`.
+
+</Concept>
+
+<Concept>
+
+## What happens to my web statistics?
+
+Your web statistics will be migrated to cPanel. You will find them under Awstats.
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:105]:
```suggestion
Your web statistics will be migrated to cPanel. You will find them under **Awstats**.
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 5]
```diff
@@ -0,0 +1,172 @@
+---
+meta:
+  title: Web Hosting Classic migration - Technical information
+  description: This page provides technical information for users having migrated their Web Hosting Classic to cPanel.
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:4]:
```suggestion
  description: This page provides technical information for users migrating their Web Hosting Classic to cPanel.
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+content:
+  h1: Web Hosting Classic migration - Technical information
+  paragraph: This page provides technical information for users having migrated their Web Hosting Classic to cPanel.
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:7]:
```suggestion
  paragraph: This page provides technical information for users migrating their Web Hosting Classic to cPanel.
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+tags: webhosting classic migration cpanel
+dates:
+  validation: 2024-01-10
+  posted: 2024-01-10
+categories:
+  - webhosting
+---
+
+You can now migrate your Online.net Web Hosting Classic plan to Scaleway's new Web Hosting platform. This migration brings you a better experience and performance. Benefits of the migration include:
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:16]:
```suggestion
You can now migrate your Online.net Web Hosting Classic plan to Scaleway's new Web Hosting platform. This migration brings you better performance and an improved user experience. Benefits of the migration include:
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+- **Up-to-date system:** cPanel's regular updates contribute to an up-to-date and secure hosting environment. This ensures that your system remains resilient in the face of evolving digital challenges.
+- **A wide range of features:** Explore a range of features offered by cPanel, covering file management, security options, email functionality, and database management. These features collectively form a powerful toolkit to address diverse hosting requirements.
+
+You can initiate the migration of your Web Hosting plan directly from your [Dedibox/Online.net console](https://console.online.net/). For further information, refer to our dedicated documentation [How to migrate an Online.net/Dedibox web hosting plan to Scaleway](/managed-services/webhosting/how-to/migrate-webhosting-plan/).
+
+<Concept opened>
+
+## What will be my new hosting plan?
+
+All migrated hosting will transferred to the **migration plan**.
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:29]: This sentence doesn't make sense to me, but I'm not sure what it should be? Is there a link to the details of the SCW plan, like showing exactly the price and featureset? Also, another question I would have is how does the user migrate?

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+    You are required to update your hosting plan to one of our regular offers once the migration of your data has been completed.
+</Message>
+
+</Concept>
+
+<Concept>
+
+## What happens to my backup option?
+
+If the Web Hosting Classic plan includes the backup option, the same option will be available on Scaleway.
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:41]:
```suggestion
If the Online.net Web Hosting Classic plan includes the backup option, the same option will be available on Scaleway Web Hosting.
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+## Where can I find my web files?
+
+All web files in the home directory will be moved to the corresponding directory in the new environment.
+
+</Concept>
+
+<Concept>
+
+Where can I find my main domain files?
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:55]:
```suggestion
## Where can I find my main domain files?
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+
+Your main domain files can be located in the `public_html` or `www` directory.
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:57]:
```suggestion
Your main domain files should be located in the `public_html` or `www` directory.
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+
+## What happens to my web statistics?
+
+Your web statistics will be migrated to cPanel. You will find them under **Awstats**.
+
+</Concept>
+
+<Concept>
+
+What happens to my web applications?
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:111]:
```suggestion
## What happens to my web applications?
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+
+## What happens to my databases?
+
+All your MySQL and PgSQL databases will be migrated to cPanel. The hostname will change and must be updated in the website's configuration files. The hostname can be set as `localhost` if the script is on the same platform or the platform's hostname or IP address.
+
+</Concept>
+
+<Concept>
+
+## What happens to my DNS Configuration (if the nameservers of the domain resolve to ns0.online.net)?
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:151]:
```suggestion
## What happens to my DNS configuration (if the nameservers of the domain resolve to ns0.online.net)?
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+
+- **Nameservers:** We will attempt to update the nameservers to `ns1.whm.fr-par.scw.cloud` and `ns2.whm.fr-par.scw.cloud`.
+- **DNS Zone:** Migration of DNS Zone records not conflicting with the new cPanel Zone will be attempted (e.g., user-custom DNS records).
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:154]:
```suggestion
- **DNS Zone:** Migration of DNS zone records not conflicting with the new cPanel zone will be attempted (e.g., user-custom DNS records).
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+
+</Concept>
+
+<Concept>
+
+When will I be billed for the migrated Web Hosting plan?
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:160]:
```suggestion
## When will I be billed for the migrated Web Hosting plan?
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+
+Your Scalewy Web Hosting billing will start either at the expiration date of Classic Hosting plan or upon the change of the offer on Scalaeway, whichever comes first.
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:162]: This isn't clear to me - what is meant by "upon to change of the offer on Scaleway" - could it be "when you migrate to Scaleway"?

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+- **Improved performance:** Migrating to cPanel can result in noticeable performance improvements. Expect more efficient resource utilization and faster loading times for your websites and applications.
+- **Up-to-date system:** cPanel's regular updates contribute to an up-to-date and secure hosting environment. This ensures that your system remains resilient in the face of evolving digital challenges.
+- **A wide range of features:** Explore a range of features offered by cPanel, covering file management, security options, email functionality, and database management. These features collectively form a powerful toolkit to address diverse hosting requirements.
+
+You can initiate the migration of your Web Hosting plan directly from your [Dedibox/Online.net console](https://console.online.net/). For further information, refer to our dedicated documentation [How to migrate an Online.net/Dedibox web hosting plan to Scaleway](/managed-services/webhosting/how-to/migrate-webhosting-plan/).
+
+<Concept opened>
+
+## What will be my new hosting plan?
```
Comment (RoRoJ) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:27]:
```suggestion
## What will my new hosting plan be?
```

**Review by RoRoJ — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 18]
```diff
@@ -0,0 +1,112 @@
...
+### What will my new hosting plan be?
+
+After [initiating the migration](/managed-services/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You will then have the opportunity to select from Scaleway's range of Web Hosting plans, tailored to meet your specific requirements.
+
+Your current offer will be transferred to one of these migration plans:
+
+| Online.net/Dedibox Web Hosting offer | Scaleway Web Hosting offer    |
+|--------------------------------------|-------------------------------|
+| Personal (€1.99/month)               | Migration (€2.99/month)       |
+| Pro (€3.99/month)                    | Lite-over-quota (€5.99/month) |
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:36]:
```suggestion
| Pro (€3.99/month)                    | Lite                     (€5.99/month) |
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 19]
```diff
@@ -0,0 +1,112 @@
...
+| Business (€5.99/month)               | Lite-over-quota (€5.99/month) |
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:37]:
```suggestion
| Business (€5.99/month)               | Lite(€5.99/month)                         |
```

File: managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+
+After [initiating the migration](/managed-services/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You will then have the opportunity to select from Scaleway's range of Web Hosting plans, tailored to meet your specific requirements.
+
+Your current offer will be transferred to one of these migration plans:
+
+| Online.net/Dedibox Web Hosting offer | Scaleway Web Hosting offer    |
+|--------------------------------------|-------------------------------|
+| Personal (€1.99/month)               | Migration (€2.99/month)       |
+| Pro (€3.99/month)                    | Lite                     (€5.99/month) |
+| Business (€5.99/month)               | Lite(€5.99/month)                         |
```
Comment (bene2k1) [inline managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx:37]:
```suggestion
| Pro (€3.99/month)                    | Lite (€5.99/month)        |
| Business (€5.99/month)               | Lite (€5.99/month)    |
```

**Pull request merged**